test/e2e/arktos/multi_tenancy/test_suites/crd/test_per_tenant_crd.yaml

```diff
@@ -0,0 +1,138 @@
+# Verifying different tenants can create their CRDs independently...
+############################################
+  - BeforeTestMessage: Verifying different tenants can create their CRDs independently...
+    Command: ${kubectl} get crds --context ${first_tenant}-admin-context
+    OutputShouldBe: "No resources found.\n"
+
+  - Command: ${kubectl} apply -f ${test_data_dir}/regular_crd.yaml --context ${first_tenant}-admin-context
+    OutputShouldBe: "customresourcedefinition.apiextensions.k8s.io/neversharingsweetdreams.fantasy.wonderland.com created\n"
+
+  - Command: ${kubectl} get crds --context ${first_tenant}-admin-context
+    OutputShouldContain:
+    - "neversharingsweetdreams.fantasy.wonderland.com"
+
+  - Command: ${kubectl} api-resources --context ${first_tenant}-admin-context
+    OutputShouldContain:
+    - "neversharingsweetdreams"
+    - "NeverSharingSweetDream"
+    - "fantasy.wonderland.com"
+
+  - Command: ${kubectl} api-versions --context ${first_tenant}-admin-context
+    OutputShouldContain:
+    - "fantasy.wonderland.com/v1\n"
+
+  - Command: ${kubectl} get crds --context ${second_tenant}-admin-context
+    OutputShouldBe: "No resources found.\n"
+
+  - Command: ${kubectl} api-resources --context ${second_tenant}-admin-context
+    OutputShouldNotContain:
+    - "neversharingsweetdreams"
+    - "NeverSharingSweetDream"
+    - "fantasy.wonderland.com"
+
+  - Command: ${kubectl} api-versions --context ${second_tenant}-admin-context
+    OutputShouldNotContain:
+    - "fantasy.wonderland.com"
+
+  - Command: "cat ${test_data_dir}/regular_crd.yaml | sed 's/version: v1/version: v1/g' |
+              ${kubectl} apply --context ${second_tenant}-admin-context -f -"
+    OutputShouldBe: "customresourcedefinition.apiextensions.k8s.io/neversharingsweetdreams.fantasy.wonderland.com created\n"
+
+  - Command: ${kubectl} get crds --context ${second_tenant}-admin-context
+    OutputShouldContain:
+    - "neversharingsweetdreams.fantasy.wonderland.com"
+
+  - Command: ${kubectl} api-resources --context ${second_tenant}-admin-context
+    OutputShouldContain:
+    - "neversharingsweetdreams"
+    - "NeverSharingSweetDream"
+    - "fantasy.wonderland.com"
+
+  - Command: ${kubectl} api-versions --context ${second_tenant}-admin-context
+    OutputShouldContain:
+    - "fantasy.wonderland.com/v1\n"
+
+  - Command: ${kubectl} api-versions --context ${first_tenant}-admin-context
+    OutputShouldContain:
+    - "fantasy.wonderland.com/v1\n"
+
+  - Command: "${kubectl} get crds --all-tenants -o json 
+            | jq -r '.items[] | [.metadata.name, .metadata.tenant] | @tsv'"
+    OutputShouldContain:
+    - "neversharingsweetdreams.fantasy.wonderland.com	${first_tenant}\n"
+    - "neversharingsweetdreams.fantasy.wonderland.com	${second_tenant}\n"
+
+
+######################################################################################################
+# Verifying different tenants can delete their custom resources independently...
```

Comment:
is this comment correct? where's the delete?

Reply:
Ah, not "delete", should be "create"